docs/apache-airflow/operators-and-hooks-ref.rst

```diff
@@ -79,7 +79,7 @@ For details see: :doc:`apache-airflow-providers:operators-and-hooks-ref/index`.
    * - :mod:`airflow.sensors.external_task`
      - :doc:`How to use <howto/operator/external_task_sensor>`
 
-   * - :mod:`airflow.sensors.filesystem`
+   * - :mod:`airflow.providers.standard.sensors.filesystem`
```

Comment:
This is not required you can remove this. because it falls under standard provider now.

Reply:
@gopidesupavan Removed!